server/runtime/src/main/java/org/infinispan/server/security/realm/EncryptedPropertiesSecurityRealm.java

```diff
@@ -271,6 +271,9 @@ public void load(InputStream usersStream, InputStream groupsStream) throws IOExc
                               String[] passwords = builder.toString().trim().split(";");
                               for (String password : passwords) {
                                  int colon = password.indexOf(':');
+                                 if (colon < 0) {
+                                    throw log.malformedUserProperties(username);
```

Comment:
I agree with Jose, maybe instead we can just keep track of what line each username is on and instead state there is a malformed user name at line number X? I am thinking if a user may have access to the server log but not write/read access to the server configuration itself.